### PR TITLE
Update the gce_run.sh to install needed Linux headers

### PR DESCRIPTION
**Pull request by johnbelamaric**

### CODE REVIEW

**Comment by johnbelamaric** [comment]
@electrocucaracha PTAL

**Comment by johnbelamaric** [comment]
So, in general this worked for me, but the repos that access gitea do not become ready, because `gitea` is not a valid host. We need to insert a service in the default namespace for that - or actually, maybe in the porch-server namespace.

But...I figure that's all going to change when we shift off the "one-summit" deployment, right? So, I didn't worry about it yet.

File: e2e/provision/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+3. On the machine, run this:
+
+```bash
+$ sudo apt-get install -y git
+$ git clone https://github.com/nephio-project/test-infra.git
+$ sed -e "s/vagrant/$USER/" < test-infra/e2e/provision/nephio.yaml > ~/nephio.yaml
+$ cd test-infra/e2e/provision/
+$ ./gce_run.sh
+$ kubectl --kubeconfig ~/.kube/mgmt-config port-forward --namespace=nephio-webui svc/nephio-webui 7007
```
Comment (electrocucaracha) [inline e2e/provision/README.md:29]: I think that this needs to be executed with `sudo`
```suggestion
$ sudo kubectl --kubeconfig ~/.kube/mgmt-config port-forward --namespace=nephio-webui svc/nephio-webui 7007
```

**Comment by electrocucaracha** [comment]
> So, in general this worked for me, but the repos that access gitea do not become ready, because `gitea` is not a valid host. We need to insert a service in the default namespace for that - or actually, maybe in the porch-server namespace.
> 
> But...I figure that's all going to change when we shift off the "one-summit" deployment, right? So, I didn't worry about it yet.

Actually, the [current](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/roles/bootstrap/tasks/main.yml#L88) version used to install gitea package is `v1`. I haven't had a chance to test v2 and use `metal LB` services.

/lgtm

File: e2e/provision/README.md (hunk 1 above)
Comment (johnbelamaric) [reply e2e/provision/README.md:29]: No, this works fine without sudo since it's not a privileged port.

Comment (electrocucaracha) [reply e2e/provision/README.md:29]: I know what happen, the `one-summit` repo is used by [default](https://github.com/nephio-project/test-infra/blob/main/e2e/provision/gce_run.sh#L48). The CI has been changed to use the `cluster-api` with the `DEPLOYMENT_TYPE` env var.

**Pull request merged**